spec/features/user_login_logout_functionality_spec.rb

```diff
@@ -16,24 +11,69 @@
       fill_in('new_user[password]', with: 'pass')
       fill_in('new_user[password_confirmation]', with: 'pass')
       click_button "Create"
-      expect(current_path).to eq(dashboard_path)
       click_link("Logout")
+    end
+
+    it "logs in the new user properly with valid info submitted" do
       click_link("Login")
-      fill_in("Email", with: "Jeffy")
+      fill_in("Email", with: 'J@gmail.com')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.